### ISSUE TITLE
Code linting and formatting

### ISSUE THREAD

**Issue by nfoert** [nfoert/cardie#73]
Implement various things to the repository to allow the code to be automatically linted and formatted
- [ ] Python with `ruff`
- [ ] JavaScript with [`biome`](https://biomejs.dev/)
- [ ] Django Templates with [`djlint`](https://www.djlint.com/)

**Comment by mSaurabh** [comment]
@ar4s are you actively working on biome implementation? If not can I take this one? Also any pointers on some of the things you want me to focus on here?

**Comment by ar4s** [comment]
Yes and change a scope of my (PR)[https://github.com/nfoert/cardie/pull/76] - I am not able to fix all errors :( I fixed only JS files in `global` directory. Futhermore, thanks to `index.css` I found the bug in `biome` itself (it not recognize `scroll-snap-points-y` property) :) so I excluded all CSS files at this moment.
@nfoert I would be very happy if you could test the following pull request and eventually merge it into development.
@mSaurabh  I would appreciate if you able to fix another module. In `biome.json` at `files.include` you can add another module to check.

**Comment by nfoert** [comment]
> I found the bug in biome itself (it not recognize scroll-snap-points-y property)

That's interesting, is that something that is reported somewhere and will be fixed eventually? Checking the biome repo [for related issues](https://github.com/biomejs/biome/issues?q=sort%3Aupdated-desc+is%3Aissue+scroll-snap+is%3Aclosed+) yielded no results

> I would be very happy if you could test the following pull request and eventually merge it into development.

I just tested it out and left a comment on #76! I think it may be wise to wait until we can format all of the JavaScript files before merging the PR

**Comment by ar4s** [comment]
> That's interesting, is that something that is reported somewhere and will be fixed eventually? Checking the biome repo [for related issues](https://github.com/biomejs/biome/issues?q=sort%3Aupdated-desc+is%3Aissue+scroll-snap+is%3Aclosed+) yielded no results

I will create an issue in `biome` afer I double check this but in 99,9% is a bug because `biome` uses database from `known-css-properties` but it is not synchronize automatically.

> I just tested it out and left a comment on https://github.com/nfoert/cardie/pull/76! I think it may be wise to wait until we can format all of the JavaScript files before merging the PR

We can wait for resolve this problem on the `biome` side and wait for official release.
